scripts/lint-lists.py

```diff
@@ -135,6 +149,11 @@ def main(source='OONI', notes='', legacy=False, fix_duplicates=False):
                     )
                 url = url.strip().lower()
                 canonical_url = url
+                if os.path.basename(csv_path) != "global.csv":
+                    if canonical_url in check_against_global:
+                        errors.append(
+                            DuplicateURLWithGlobalList(canonical_url, csv_path, idx+2)
+                        )
```

Comment:
I would suggest doing this check against the "canonical_url" and storing in the `global_urls_bag` the canonical representation of the url (this way we will detect dupes when we have two URLs that are `http://foo.com/` & `http://foo.com`).